spi/imx6ull-qspi/qspi.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Phoenix-RTOS
+ *
+ * i.MX 6ULL QSPI lib API
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Hubert Badocha
+ *
+ * %LICENSE%
+ */
+
+#ifndef IMX6ULL_QSPI_API_H
+#define IMX6ULL_QSPI_API_H
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include <sys/threads.h>
+#include <phoenix/arch/imx6ull.h>
```

Comment:
I don't think these ones are needed in this header. Header should provide an absolute minimum of additional headers.
```suggestion
#include <sys/threads.h>
#include <phoenix/arch/imx6ull.h>
```